swap_meet/item.py

```diff
@@ -1,2 +1,29 @@
+from uuid import uuid4
+from .constants import CONDITIONS
+from.errors import InvalidIDError
 class Item:
-    pass
\ No newline at end of file
+    """
+    NOTE:
+    Raises: InvalidIDError: If the provided ID is not an integer.
+    """
+
+    def __init__(self, id=None, condition=0, age=None):
+        self.id = int(uuid4()) if id is None else id
```

Comment:
👍 The `Item` constructor properly sets its attributes and allows for default parameters. Similar to needing new lists in the Vendor constructor, here, we need to run a function for the default case, which means that we need a sentinel to mark that no value was supplied so that we can then run the id generation logic within the constructor, ensuring that every item gets its own id.